presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/SimplifyCountOverConstant.java

```diff
@@ -77,7 +77,9 @@ public Pattern getPattern()
                 aggregations.put(symbol, new AggregationNode.Aggregation(
                         new FunctionCall(QualifiedName.of("count"), ImmutableList.of()),
                         new Signature("count", AGGREGATE, parseTypeSignature(StandardTypes.BIGINT)),
-                        aggregation.getMask()));
+                        aggregation.getMask(),
+                        aggregation.getOrderBy(),
```

Comment:
I thought about it. Figured I'll leave removing order by to a separate optimization all together. Most aggregations are not order sensitive anyways.